worlds/oot/__init__.py

```diff
@@ -1173,10 +1173,33 @@ def hint_type_players(hint_type: str) -> set:
 
     def fill_slot_data(self):
         self.collectible_flags_available.wait()
-        return {
+
+        slot_data = {
             'collectible_override_flags': self.collectible_override_flags,
             'collectible_flag_offsets': self.collectible_flag_offsets
         }
+        slot_data.update(self.options.as_dict("open_forest", "open_kakariko", "open_door_of_time", "zora_fountain", "gerudo_fortress",
```

Comment:
A lot of these lines are really long, much longer than 120 characters, the linebreaks also seem to be pretty much random. I think you should make the lines indented to this level and then also organize them more
```suggestion
        slot_data.update(self.options.as_dict(
            "open_forest", "open_kakariko", "open_door_of_time", "zora_fountain", "gerudo_fortress",
```